src/main/com/fulcrologic/guardrails/core.cljc

```diff
@@ -70,16 +99,19 @@
                             "\n"
                             fn-name
                             (if args? " argument list" " return type") "\n"
-                            problem)]
+                            problem)
+              context     (get-global-context)]
           (if throw?
             (reset! valid-exception
-              (ex-info description
+              (ex-info (cond->> description context
+                         (str "\nContext: " context))
```

Comment:
I would prefer cond-> be formatted like cond. Putting the condition on the line with the thing being threaded is confusing IMO